pkg/cm_accounts/cm_accounts.go

```diff
@@ -77,6 +86,18 @@ type Service interface {
 		price *big.Int,
 		paymentToken common.Address,
 	) (*types.Receipt, error)
+
+	getCurrentImplementationOnManager(cmAccountAddress common.Address) (common.Address, error)
```

Comment:
no need for those to be added to interface, since they're private and will only be called from service itself